### PR TITLE
Upgrade leapc

### PR DESCRIPTION
**Pull request by Amarcolina**
- Change api usage to match new api, only small changes like removing IsEmbedded
 - Added a better unit test to make sure that TransformCopy produces a binary equivalent frame when using the identity transform.  Should help copy errors in the future.

NOTE: this PR does not yet contain the updated dlls.  This PR is to test with 6791 on platform.  Once 6791 is validated, use that branch to build the dlls to insert into this PR before it is merged.

### CODE REVIEW

File: Assets/LeapMotion/Scripts/LeapServiceProvider.cs [hunk 1]
```diff
@@ -142,7 +142,7 @@ public class LeapServiceProvider : LeapProvider {
 
         // TODO: Add baseline & offset when included in API
         // NOTE: Alternative is to use device type since all parameters are invariant
-        info.isEmbedded = devices[0].IsEmbedded;
+        info.isEmbedded = devices[0].Type == Device.DeviceType.TYPE_KEYBOARD || devices[0].Type == Device.DeviceType.TYPE_LAPTOP;
```
Comment (jo3w4rd) [inline Assets/LeapMotion/Scripts/LeapServiceProvider.cs]: Note that device type isn't actually implemented. But either way, since the service no longer has a concept mapping to "isEmbedded" we might just want to remove it here.

**Comment by jselstad** [comment]
On hold until Image Warping regression is figured out.

File: Assets/LeapMotion/Scripts/LeapServiceProvider.cs (hunk 1 above)
Comment (Amarcolina) [inline Assets/LeapMotion/Scripts/LeapServiceProvider.cs]: That sounds good to me.  I guess there should be a revamp for device info at some point, so we can stop hacking around, but I think that all of this should be in a separate PR, since it needs wider changes like fixing Disconnection Notice

**Comment by Amarcolina** [comment]
This PR is blocked for the moment.  The layout of LEAP_HAND changed, breaking IE

**Comment by Amarcolina** [comment]
Closing this until resolved
